A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_AirportsMonitor.js

```diff
@@ -0,0 +1,194 @@
+class CDUAirportsMonitor {
+    static ShowPage(mcdu, reset=false) {
```

Comment:
```suggestion
    static ShowPage(mcdu, reset = false) {
```

Reply:
Ah yes, I'm coming from mostly Python at the moment ;)